packages/terra-application-navigation/tests/jest/drawer-menu/DrawerMenu.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import DrawerMenu from '../../../src/drawer-menu/_DrawerMenu';
+import mockIntl from '../mockIntl';
```

Comment:
mockIntl is provided in the terra-enzyme-intl package: https://github.com/cerner/terra-enzyme-intl#mockintl

Reply:
Updated.